http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AzurePubsubHandler.scala

```diff
@@ -463,7 +462,7 @@ class AzurePubsubHandlerInterp[F[_]: Parallel](
             val hostIp = s"${params.relayNamespace.value}.servicebus.windows.net"
 
             for {
-              _ <- clusterQuery.updateClusterHostIp(params.runtime.id, Some(IP(hostIp)), ctx.now).transaction
+              _ <- clusterQuery.updateClusterHostIp(params.runtime.id, Some(IP(hostIp)), Instant.now).transaction
```

Comment:
Nit: since `Instant.now` is technically a side effect, we normally wrap it in an `F` context. You can do something like:
```
for {
  now <- nowInstant
  _ <- clusterQuery.updateClusterHostIp(params.runtime.id, Some(IP(hostIp)), now).transaction
  …
} yield ()
```
…which should still have the effect of getting the current instant.

Reply:
This is definitely not second nature yet-- I appreciate the tip :)